src/renderer/pages/Project.tsx

```diff
@@ -167,6 +168,11 @@ const ProjectPage = ({ containerRef }: Props) => {
     }
   };
 
+  const onScrubberChange = (newTime: number) => {
+    const nSeconds = newTime - time;
+    videoPreviewControllerRef.current?.seekNSeconds(nSeconds);
```

Comment:
Why do we need `seekNSeconds`? Can't you just `setPlaybackTime`? Otherwise you're making an absolute time relative, and then making that relative time absolute again which is a bit redundant

Reply:
Good catch. This now just calls setPlaybackTime with the new time returned by the scrubber.